dangerfile.js

```diff
@@ -21,20 +19,11 @@ if (title_type) {
 sourcefiles.forEach(function(filename) {
   const additions = danger.git.diffForFile(filename)
   additions.then(diff => {
-    const issues = []
     diff.added.split("\n").forEach(function(item) {
       if (item.includes("TODO:")) {
-        const has_issue = item.match(ISSUE_REGEX)
-        if (!has_issue) {
-          fail(`Source file ${filename} includes a TODO with no Mudlet issue link.\n  New TODO items in source files must have an accompanying github issue`)
-        } else {
-          issues.push(has_issue[1])
-        }
+          fail(`Source file ${filename} includes a TODO! Can you make the change right away? If no, better not to create a TODO - they just tend to hang around and never get fixed.`)
```

Comment:
The only thing is that in this sort of situation it likely means that a further PR will be necessary OR an existing PR needs to be bigger to do so - yet I am sometimes being brought up by other Core Devs on the scale of a PR because I try to do too much at once...

:man_shrugging:

Reply:
Fair call, the 'mega PRs' still applies here to keep the scopes reasonable - so another PR would be better indeed.